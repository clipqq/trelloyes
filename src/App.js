import React from 'react';
import './App.css';
import Lists from './Lists.js'

function App(props) {

  const testTitle = 'To-Do'
  const testCards =
    [
      {
        id: 3463,
        title: 'cliff',
        task: 'do something'
      },
      {
        id: 5749,
        title: 'willie',
        task: 'do something else'
      }
    ]

  /////////////////////////////
  return (
    <div className="App">
      <Lists header={testTitle} cardStore={testCards}/>
    </div>
  )
}


export default App;
