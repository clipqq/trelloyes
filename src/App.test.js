import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';


describe('<App />', () => {
  // it('renders without crashing', () => {
  //   const div = document.createElement('div');
  //   ReactDOM.render(<App />, div);
  //   ReactDOM.unmountComponentAtNode(div);
  // });

  it('renders test cards', () => {
    const testTitle = 'To-Do'
    const testCards =
      [
        {
          id: 3463,
          title: 'cliff',
          task: 'do something'
        },
        {
          id: 5749,
          title: 'willie',
          task: 'do something else'
        }
      ]
      const div = document.createElement('div');
      ReactDOM.render(<App userHeader={testTitle} userCards={testCards}/>, div);
      ReactDOM.unmountComponentAtNode(div);
  })

  it('matches snapshot', () => {
    // Render the component, as JSON
    const tree = renderer.create(<App />).toJSON();
    // Check whether it matches the previous snapshot
    // Stored in __snapshots__/App.test.js.snap
    expect(tree).toMatchSnapshot();
  });
});
