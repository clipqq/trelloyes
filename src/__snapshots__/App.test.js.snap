// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches snapshot 1`] = `
<div
  className="App"
>
  <div
    class="List-cards"
  >
    <div
      className="card"
    >
      <h3>
        cliff
      </h3>
      <p>
        do something
      </p>
    </div>
  </div>
  <div
    class="List-cards"
  >
    <div
      className="card"
    >
      <h3>
        willie
      </h3>
      <p>
        do something else
      </p>
    </div>
  </div>
</div>
`;
